src/c++/perf_analyzer/test_load_manager_base.h

```diff
@@ -106,6 +106,48 @@ class TestLoadManagerBase {
       }
     }
   }
+  // FIXME: remove the print statements
+  //
+  void CheckSharedMemory()
+  {
+    auto stats = GetStats();
+
+    if (params_.shared_memory_type == SYSTEM_SHARED_MEMORY) {
```

Comment:
Should any of these that are ">0" be "==1" ?

Reply:
Also, the need to allow values >0 instead of exact values might kill my idea above to make this a struct and just compare the structs

Reply:
Updated to == 1. This still works with the struct suggestion above.